src/main/java/org/jenkinsci/extension_indexer/ExtensionPointListGenerator.java

```diff
@@ -302,10 +306,10 @@ private void generateAsciidocReport() throws IOException {
             value.add(f);
         }
 
-        asciidocOutputDir.mkdirs();
+        Files.createDirectories(asciidocOutputDir.toPath());
 
-        try (PrintWriter w = new PrintWriter(new File(asciidocOutputDir, "index.adoc"))) {
-            IOUtils.copy(new InputStreamReader(getClass().getResourceAsStream("index-preamble.txt")), w);
+        try (PrintWriter w = new PrintWriter(new File(asciidocOutputDir, "index.adoc"), "UTF-8")) {
+            IOUtils.copy(new InputStreamReader(getClass().getResourceAsStream("index-preamble.txt"), "UTF-8"), w);
```

Comment:
Ditto:

```suggestion
        try (PrintWriter w = new PrintWriter(new File(asciidocOutputDir, "index.adoc"), StandardCharsets.UTF_8)) {
            IOUtils.copy(new InputStreamReader(getClass().getResourceAsStream("index-preamble.txt"), StandardCharsets.UTF_8), w);
```

Reply:
Resolved in 48046e678b43302ab5192837ca26cbf36f6ad865